<launch>
  <arg name="rviz_config" default="$(find polygon_coverage_ros2)/cfg/rviz/coverage_planner.rviz"/>

  <node name="rviz" pkg="rviz" type="rviz" output="log" args="-d $(arg rviz_config)"/>

  <node name="coverage_planner" pkg="polygon_coverage_ros2" type="coverage_planner" output="screen" clear_params="true">

    <rosparam command="load" file="$(find polygon_coverage_ros2)/cfg/coverage_planner.yaml"/>
    <!-- Polygon, altitude, and world frame ID -->
    <rosparam command="load" file="$(find polygon_coverage_ros2)/cfg/polygons/example_polygon_epfl_simple.yaml"/>

  </node>

</launch>
